ActindoWorkFlow/UserInterface/03a_Processes.md

```diff
@@ -2,42 +2,20 @@
 [!!Configure the queue types](../Integration/01_ConfigureQueueTypes.md)
 [!!Process actions](./04a_ProcessActions.md)
 
-# Overview
+# OVERVIEW (Processes)
 
-*Workflows > Processes > Tab OVERVIEW*
+*Process Orchestration > Processes > Tab OVERVIEW*
 
 ![Overview](../../Assets/Screenshots/ActindoWorkFlow/Processes/Processes.png "[Overview]")
 
 **Processes**
 
-- *VIEW*  
-    Click the drop-down list to select the view. All created views are displayed in the drop-down list. Click the ![Points](../../Assets/Icons/Points01.png "[Points]") (Points) button to the right of the *VIEW* drop-down list to display the context menu and create a view.   
-
-    - ![Points](../../Assets/Icons/Points01.png "[Points]") (Points)      
-        Click this button to the right of the *View* drop-down list to display the context menu. The following menu entries are available:
-
-      - ![Create](../../Assets/Icons/Plus06.png "[Create]") create  
-          Click this entry to create a view. The *Create view* window is displayed, see [Create view](#create-view).
-
-      - ![Rename](../../Assets/Icons/Edit02.png "[Rename]") rename  
-          Click this entry to rename the selected view. The *Rename view* window is displayed, see [Rename view](#rename-view). This menu entry is only displayed if a view has been selected.
-
-      - ![Reset](../../Assets/Icons/Reset.png "[Reset]") reset  
-          Click this entry to reset all unsaved changes to the settings of the selected view. This menu entry is only displayed if a view has been selected and any changes have been made to the view settings.
-
-      - ![Publish](../../Assets/Icons/Publish.png "[Publish]") publish  
-          Click this entry to publish the view. This menu entry is only displayed if a view has been selected and unpublished.
-
-      - ![Unpublish](../../Assets/Icons/Unpublish.png "[Unpublish]") unpublish  
-          Click this entry to unpublish the view. This menu entry is only displayed if a view has been selected and published.
-
-      - ![Save](../../Assets/Icons/Save.png "[Save]") save  
-          Click this entry to save the current view settings in the selected view. This menu entry is only displayed if a view has been selected.
+The list displays all processes that are matching the selected values (status and workflow type). Depending on the settings, the displayed columns may vary. All fields are read-only. Click a process in the list to display the *Process ID* view with a process diagram and detailed information about the selected process.
```

Comment:
Vorschlag: The list displayes all processes available in the system/currently available. You can filter the processes to be displayed by status and workflow type... (oder so was). Prinzipiell sind Status und Workflow type Filters nicht eingesetzt, daher würde man alle sehen.